waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/ThriftMetastoreClientManager.java

```diff
@@ -53,7 +53,7 @@
   private ThriftHiveMetastore.Iface client = null;
   private TTransport transport = null;
   private boolean isConnected = false;
-  private URI metastoreUris[];
+  private URI[] metastoreUris;
```

Comment:
There are multiple styles of array type definitions in Java: 
* Java style: public static void main(String[] args)
* C style: public static void main(String args[])

The Checkstyle configuration opts for the first one by default.

Reply:
I'm not sure how this version ever slipped in!